### PR TITLE
[Modal ] Modal class attribute fix

### PR DESCRIPTION
**Pull request by skumar09**
PR includes a fix for the following 

- Modal class attribute update  from: ` class: 'modal link-block'` To ` class: 'modal link-block '`

### CODE REVIEW

**Review by JackySun9 — APPROVED** [review]

File: selectors/milo/modal.block.page.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class Modal {
     // modal contents attributes
     this.attProperties = {
       'modal-link': {
-        class: 'modal link-block',
+        class: 'modal link-block ',
```
Comment (amauch-adobe) [inline selectors/milo/modal.block.page.js:21]: All that was added here was a space is it necessary?

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Just curious about the extra space.

File: selectors/milo/modal.block.page.js (hunk 1 above)
Comment (skumar09) [reply selectors/milo/modal.block.page.js:21]: @amauch-adobe: That was the change, (as per my earlier experience, I have seen having space, but not sure here why this is introduced...)

**Review by amauch-adobe — APPROVED** [review]
Approved

**Pull request merged**